market/market-promo-text-sv.txt

```diff
@@ -1 +1 @@
-Dina egna dash, i din egen ficka!
\ No newline at end of file
+Your own Dash, in your own pocket!
```

Comment:
All of the market promos are changed from translated to English versions. Not sure if this is a good idea. Where is it used exactly?

Reply:
This is not used anywhere in the app.  Text Files have to be manually imported into the Google Play Store listings.  The promo-text file probably wasn't used by us at any time.

The other text files are more concerning.  

Also, there are some empty XML files.  

I wonder if something changed with the Transifex Client.  Is it working differently?  Why would we be getting English in what should be translated in other languages?

Reply:
I took all of these changes out since they were from inactive translations.